packages/attribute-slicer-powerbi/src/dataConversion.ts

```diff
@@ -67,13 +67,13 @@ export default function converter(
         dvCats: powerbi.DataViewCategoryColumn[],
         catIdx: number,
         total: number,
-        id: powerbi.visuals.ISelectionId,
+        selectionid: powerbi.visuals.ISelectionId,
         valueSegments: IValueSegment[],
       ) => {
         const item = createItem(
           buildCategoryDisplay(dvCats, catIdx, categoryFormatter),
           total,
-          id.getKey ? id.getKey() : <any>id,
+          dvCats[0].values[catIdx] + '',
```

Comment:
What is adding the empty string here for?

Reply:
Coerces it into a string is all